src/modules/translator.py

```diff
@@ -0,0 +1,87 @@
+import deepl
+import googletrans
+from os import getenv
+from dotenv import load_dotenv
+from google.cloud import translate
+load_dotenv()
+
+TRANSLATOR = getenv('TRANSLATOR', 'google_web')
+DEEPL_AUTH_KEY = getenv('DEEPL_AUTH_KEY')
+GOOGLE_PROJECT_ID = getenv('GOOGLE_PROJECT_ID')
+
+class Translator:
+    def __init__(self, text="Hello, world!",source="en",target="ja"):
+        self.mode = TRANSLATOR
+        self.text = text
+        self.source = source
+        self.target = target
+        self.interpreter = None
+        self.google_project_id=GOOGLE_PROJECT_ID
+        self.deepl_auth_key=DEEPL_AUTH_KEY
+
+    def get_source(self):
+        return self.source
+
+    def get_target(self):
+        return self.target
+    
+    def get_text(self):
+        return self.text
+
+    def get_mode(self):
+        return self.mode
+
+    def set_text(self, text):
+        self.text = text
+
+    def set_source(self, source):
+        self.source = source
+    
+    def set_target(self, target):
+        self.target = target
+    
+    def set_mode(self, mode):
+        self.mode = mode
```

Comment:
Seems this could be converted into a `dataclass` and would preform the same actions. 

https://docs.python.org/3/library/dataclasses.html

Instead of all these setters and getters, you could do:

```python
from dataclasses import dataclass

@dataclass
class Translator:
    text: str = "Hello, world!"
    source: str = "en"
    target: str = "ja"
    mode: str = "TRANSLATOR"
    interpreter: str = None
    google_project_id: str = GOOGLE_PROJECT_ID
    deepl_auth_key: str = DEEPL_AUTH_KEY
```

Then you can access these values the same way as any other.

```
def some_func(self):
    if self.source == self.target:
        raise ValueError("Source and Target is the same")
    return self.translate()
```

*Just as an example*